rioting-daisies/BangazonWorkforce
# Created View Model For listing all Departments

# Description
Create Get all Department Method in DepartmentController to display The Name, Budget, and number of employees in each department. You can access this by clicking the departments link on the homepage.



Fixes Issue # https://github.com/rioting-daisies/BangazonWorkforce/issues/4

## Type of change

Please delete options that are not relevant.


- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions

Git fetch --all and checkout cm-department-viewModel
run bangazon and click on the link for departments
Ensure all departments and there correct information are displayed.



# Checklist:
